data/tlg0062/tlg001/tlg0062.tlg001.perseus-eng1.xml

```diff
@@ -0,0 +1,683 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">Phalaris</title>
+<author>Lucian of Samosata</author>
+<editor>Austin Morris Harmon, 1878-1950</editor>
+<funder>National Endowment for the Humanities</funder>
+<principal>Gregory Crane</principal>
```

Comment:
I add the staff here who were working on texts when the files were created based on the initial year of upload

Reply:
I assume these are the same as the Greek, but can look this up